templates/.rubocop.yml

```diff
@@ -7,50 +7,14 @@ Metrics/LineLength:
 Style/Documentation:
   Enabled: false
 
-Style/NonNilCheck:
-  IncludeSemanticChanges: true
-
-Naming/PredicateName:
-  Exclude:
-    - 'app/serializers/**/*'
-
-Metrics/BlockLength:
-  Exclude:
-  - config/initializers/simple_form_bootstrap.rb
-  - 'Rakefile'
-  - '**/*.rake'
-  - 'spec/**/*.rb'
-
-Naming/FileName:
-  Exclude:
-  - 'Gemfile'
-
 Style/FrozenStringLiteralComment:
   Enabled: false
-  
-Style/FormatStringToken:
-  Enabled: false
 
-Rails/LexicallyScopedActionFilter:
-  Enabled: false
-
-Rails/SkipsModelValidations:
+Style/MixinUsage:
   Exclude:
-    - 'spec/**/*.rb'
+  - 'bin/setup'
+  - 'bin/update'
```

Comment:
With the rules until here, the standard Rails 5.2 setup is green (after autocorrect). Now follow our more custom rules.